PhotoLocatorTest/Helpers/VideoTransformsTest.cs

```diff
@@ -14,7 +14,7 @@ public class VideoTransformsTest
         [TestMethod]
         public async Task RunFFmpegWithStreamOutputImagesAsync_ShouldProcessImages()
         {
-            if (string.IsNullOrEmpty(FFmpegPath))
+            if (string.IsNullOrWhiteSpace(FFmpegPath))
```

Comment:
This is a const in the test code, so we don't need to check if it is whitespace only

Reply:
I agree, but then we wouldn't need to check it at all. BUT I think a check is still useful.

Checking for whitespaces will come in handy if the constant will be moved to the UI for configuration.